File: OdonSys-SPA/src/app/core/models/procedure/procedure-model.ts
export class ProcedureModel {
  constructor(
    public id: string,
    public active: boolean,
    public dateCreated: Date,
    public dateModified: Date,
    public name: string,
    public description: string,
    public procedureTeeth: string[],
    public price: number,
    public xRays: boolean
  ) {}
}

File: OdonSysBackEnd/Resource/Contract/Contract.Authentication/Procedures/ProcedureModel.cs
namespace Contract.Workspace.Procedures;

public record ProcedureModel
(
    string Id,
    bool Active,
    DateTime DateCreated,
    DateTime DateModified,
    string Name,
    string Description,
    int Price,
    IEnumerable<string> ProcedureTeeth,
    bool XRays
);
